server/http.go

```diff
@@ -544,7 +438,7 @@ func getString(key string, props model.StringInterface) string {
 }
 
 func (p *Plugin) deauthorizeUser(w http.ResponseWriter, r *http.Request) {
-	if !p.verifyWebhookSecret(r) {
+	if !p.verifyMattermostWebhookSecret(r) {
 		http.Error(w, "Not authorized", http.StatusUnauthorized)
 		return
 	}
```

Comment:
After renaming this called method and durfaced this change, it seems odd to me that we are using a webhook secret for a `deauthorizeUser` operation. I'm not sure if this route gets called

Reply:
It's an optional feature of the plugin:

https://github.com/mattermost/mattermost-plugin-zoom/blob/229517fa473f58f0792ea6ef735ea0b2c838359e/docs/installation/zoom-configuration/zoom-setup-oauth.md#L44-L53

Reply:
agree on doing that in a separate effort, pls create a Jira ticket for that and add a comment close to the related code

Reply:
Created ticket here https://github.com/mattermost/mattermost-plugin-zoom/issues/291

Reply:
Thanks! Could you please add a comment to `deauthorizeUser` with the link to the ticket?